oxide/lib_test.go

```diff
@@ -144,3 +155,58 @@ func Test_buildRequest(t *testing.T) {
 		})
 	}
 }
+
+func Test_Client(t *testing.T) {
```

Comment:
Just for convention the test name should be Test_{name of function we're testing}
```suggestion
func Test_NewClient(t *testing.T) {
```